packages/tenant-process/test/db.test.ts

```diff
@@ -881,6 +881,108 @@ describe("database test", async () => {
+      const tenant5: Tenant = {
+        ...mockTenant,
+        id: uuidv4(),
+        name: "A tenant5",
+      };
+      it("should get tenants", async () => {
+        await addOneTenant(tenant1, postgresDB, tenants);
+        await addOneTenant(tenant2, postgresDB, tenants);
+        await addOneTenant(tenant3, postgresDB, tenants);
+
+        const tenantsByName = await readModelService.getTenantsByName({
+          name: undefined,
+          offset: 0,
+          limit: 50,
+        });
+        expect(tenantsByName.totalCount).toBe(3);
+        expect(tenantsByName.results).toEqual([tenant1, tenant2, tenant3]);
+      });
+      it("should get tenants by name", async () => {
+        await addOneTenant(tenant1, postgresDB, tenants);
+
+        await addOneTenant(tenant2, postgresDB, tenants);
+
+        const tenantsByName = await readModelService.getTenantsByName({
+          name: "A tenant1",
+          offset: 0,
+          limit: 50,
+        });
+        expect(tenantsByName.totalCount).toBe(1);
+        expect(tenantsByName.results).toEqual([tenant1]);
+      });
+      it("should not get tenants if there are not any tenants", async () => {
+        const tenantsByName = await readModelService.getTenantsByName({
+          name: undefined,
+          offset: 0,
+          limit: 50,
+        });
+        expect(tenantsByName.totalCount).toBe(0);
+        expect(tenantsByName.results).toEqual([]);
+      });
+      it("should not get tenants if the name does not match", async () => {
+        await addOneTenant(tenant1, postgresDB, tenants);
+
+        await addOneTenant(tenant2, postgresDB, tenants);
+
+        const tenantsByName = await readModelService.getTenantsByName({
+          name: "A tenant6",
+          offset: 0,
+          limit: 50,
+        });
+        expect(tenantsByName.totalCount).toBe(0);
+        expect(tenantsByName.results).toEqual([]);
+      });
+      it("Should get tenants (pagination: limit)", async () => {
+        await addOneTenant(tenant1, postgresDB, tenants);
+        await addOneTenant(tenant2, postgresDB, tenants);
+        await addOneTenant(tenant3, postgresDB, tenants);
+        await addOneTenant(tenant4, postgresDB, tenants);
+        await addOneTenant(tenant5, postgresDB, tenants);
+        const tenantsByName = await readModelService.getTenantsByName({
+          name: undefined,
+          offset: 0,
+          limit: 4,
+        });
+        expect(tenantsByName.results.length).toBe(4);
+      });
+      it("Should get tenants (pagination: offset, limit)", async () => {
+        await addOneTenant(tenant1, postgresDB, tenants);
+        await addOneTenant(tenant2, postgresDB, tenants);
+        await addOneTenant(tenant3, postgresDB, tenants);
+        await addOneTenant(tenant4, postgresDB, tenants);
+        await addOneTenant(tenant5, postgresDB, tenants);
+        const tenantsByName = await readModelService.getTenantsByName({
+          name: undefined,
+          offset: 4,
```

Comment:
```suggestion
          offset: 2,
```
Better to have a different number from `limit` in order to make this test easier to read